### ISSUE TITLE
Linear gradients + background style error

### ISSUE THREAD

**Issue by VinceBT** [necolas/react-native-web#298]
Hello, I'm trying to implement this package for web https://github.com/react-native-community/react-native-linear-gradient
In order to use linear gradients in web I have to use the "background" attribute of the div but I get this error: 
![image](https://cloud.githubusercontent.com/assets/1362505/21458449/b5150d68-c936-11e6-8858-faf1165d26e6.png)

**Environment (include versions)**

Browser: Chrome
React Native for Web (version): 0.0.60
React (version): 15.3.2

**Comment by necolas** [comment]
Use `background-image` instead. Vendor prefixes are automatically handled

**Comment by dsernst** [comment]
In case anyone else needs this, here's a working example to replace [\<LinearGradient\>](https://github.com/react-native-community/react-native-linear-gradient) for the DOM:

```jsx
import React from 'react'
import { View } from 'react-native'

const LinearGradient = ({ style, children, colors }) => (
  <View
    children={children}
    style={{ ...style,
      backgroundImage: `linear-gradient(to bottom, ${colors[0]} 0%, ${colors[1]} 100%)`,
    }}
  />
)

LinearGradient.propTypes = {
  children: React.PropTypes.arrayOf(React.PropTypes.node),
  colors: React.PropTypes.arrayOf(React.PropTypes.string).isRequired,
  style: React.PropTypes.shape({}),
}

export default LinearGradient
```

**Comment by VinceBT** [comment]
Here the solution I've been working with:
```jsx
import React, { Component, PropTypes } from 'react';
import { View } from 'react-native';

export default class LinearGradient extends Component {

  static propTypes = {
    start: PropTypes.shape({ x: PropTypes.number, y: PropTypes.number }).isRequired,
    end: PropTypes.shape({ x: PropTypes.number, y: PropTypes.number }).isRequired,
    locations: PropTypes.arrayOf(PropTypes.number),
    colors: PropTypes.arrayOf(PropTypes.string).isRequired,
    children: PropTypes.oneOfType([PropTypes.node, PropTypes.arrayOf(PropTypes.node)]),
  };

  render() {
    const { start, end, locations, colors, style, children, ...otherProps } = this.props;
    const vec = [end.x - start.x, -(end.y - start.y)];
    const angleRad = Math.atan(vec[1] / vec[0]);
    const angleDeg = Math.round((angleRad * 180) / Math.PI) +180;
    const realLocations = locations || colors.map((color, i) => (1 / (colors.length - 1)) * i);
    const colorStrings = colors.map((color, i) => `${color} ${Math.round(realLocations[i] * 100)}%`).join(', ');
    return (
      <View
        {...otherProps}
        style={[style, { backgroundImage: `linear-gradient(${angleDeg}deg, ${colorStrings})` }]}>
        {children}
      </View>
    );
  }
}
```

**Comment by leon3s** [comment]
If you use expo and need component with same props, you can look a this :)
https://gist.github.com/leon3s/a53ef0965e7c24a739d7adbb4946d97a

**Comment by VinceBT** [comment]
[They already have one](https://docs.expo.io/versions/latest/sdk/linear-gradient.html)

**Comment by VinceBT** [comment]
Seems nice butit misses the ```locations``` prop support

**Comment by passabilities** [comment]
This doesn't seem to be working..

**Comment by DninoAdnane** [comment]
I found a solution, what I did, I add it in css page, then inspect the element.
For example : for * background: linear-gradient(to right, #000 0%,  #000 50%, #fff 50% #fff 100%)*; if you inspect it using inspect element, you will find : *-webkit-gradient(linear, left top, right top, from(#000), color-stop(50%, #000), color-stop(50%, #fff))* .
So just add this last line in the inline style of your react compenent:

    <div className="card" style={{background:"-webkit-gradient(linear, left top, right top, from(#000), color-stop(50%, #000), color-stop(50%, #fff))"}}/>

for my example, I used this one above instead of : 

    <div className="card" style={{background: linear-gradient(to right, #000 0%,  #000 50%, #fff 50% #fff 100%)"}}/>

So in conclusion, add this to a css file, it will work, see the output in the "inspect element", and use it and edit it as you like in the inline style of your react js component.
